src/systems/logical_audio_sensor_plugin/LogicalAudio.hh

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#ifndef IGNITION_GAZEBO_SYSTEMS_LOGICAL_AUDIO_SENSOR_PLUGIN_LOGICALAUDIO_HH_
+#define IGNITION_GAZEBO_SYSTEMS_LOGICAL_AUDIO_SENSOR_PLUGIN_LOGICALAUDIO_HH_
+
+#include <string>
+
+#include <ignition/gazebo/components/LogicalAudio.hh>
+#include <ignition/gazebo/config.hh>
+#include <ignition/math/Pose3.hh>
+
+namespace ignition
+{
+namespace gazebo
+{
+// Inline bracket to help doxygen filtering.
+inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
+namespace logical_audio
+{
+  /// \brief Determines if an audio device can detect volume at a certain level.
+  /// \param[in] _volumeLevel The volume level that the microphone is
+  /// attempting to detect. This should be a value between 0.0 (no volume) and
+  /// 1.0 (maximum volume).
+  /// \param[in] _volumeDetectionThreshold The listening device's minimum audio
+  /// detection volume. This should be a value between 0.0 and 1.0. A device
+  /// with a low detection threshold can hear a wider range of audio than a
+  /// device with a higher detection threshold.
+  /// \return true if the listening device can detect volume at _volumeLevel,
+  /// false otherwise.
+  bool detect(const double _volumeLevel,
+      const double _volumeDetectionThreshold);
```

Comment:
we're starting to omit `const` from primitive types like `double` and also `bool` below in `computeVolume`

Reply:
Fixed in https://github.com/ignitionrobotics/ign-gazebo/pull/401/commits/0475b3dc0fd6e120247fe19ad15a38b9b78d1ca7